### PR TITLE
Hanan/acceptTAndC

### PR DESCRIPTION
**Pull request by HananINouman**

### CODE REVIEW

File: Lido-Script/invites.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+cluster_name,invite
+Lido x Obol: Luminous Lynx,https://dev.launchpad.obol.tech/dv?configHash=0xb45856248782a380137ffe7c3d067ee8b520c3d75ef4e8af7aa4dec10f44ff38
+Lido x Obol: Quaint Quokka,https://dev.launchpad.obol.tech/dv?configHash=0x8f8bf2052bf6badf130e3d00b77c1b4c1cb824b21971ebfe2ec00fd175ac2c89
```
Comment (LukeHackett12) [inline Lido-Script/invites.csv]: are these needed?

File: Lido-Script/script.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ const signer = wallet.connect(null);
 
 // Setup SDK for mainnet, make sure to change withdrawal credentials and fee recipient if you change the network/chainId!
 const client = new Client(
-  { baseUrl: "https://api.obol.tech", chainId: 1 },
+  { baseUrl: "https://api.obol.tech", chainId: 17000 },
```
Comment (LukeHackett12) [inline Lido-Script/script.js]: should the lido script have an argument in it for mainnet/holesky instead of this switching?

**Review by LukeHackett12 — APPROVED** [review]
lgtm just a few questions

File: Lido-Script/invites.csv (hunk 1 above)
Comment (HananINouman) [reply Lido-Script/invites.csv]: no sorry will delete the file

**Pull request merged**